tests/compilation_tests/database_test.py

```diff
@@ -12,39 +12,25 @@
 
 class TestDataBase(unittest.TestCase):
     # DataBase instances to test on
-    robot_state_default = Robot_State(x_pos=0, y_pos=0, heading=0, epsilon=0, max_velocity=0,
+    robot_state_default = Robot_State(xpos=0, ypos=0, heading=0, epsilon=0, max_velocity=0,
                           radius=0)
     robot_default = Robot(robot_state=robot_state_default)
 
     db_default = DataBase(robot_default)
-
+    
     # We can't make is_sim false because it fails GitHub merge tests.
-    robot_state_initial = Robot_State(x_pos=0, y_pos=0, heading=0, epsilon=0, max_velocity=0,
-                          radius=0)
-    robot_state_initial.plastic_level = 2
-    robot_state_initial.move_dist = 0.6
-    robot_state_initial.position_noise = 0.23
-    robot_state_initial.position_kp = 0.12
-    robot_state_initial.position_ki = 1.7
-    robot_state_initial.position_kd = 9.2
-    robot_state_initial.heading_kp = 0.2
-    robot_state_initial.heading_ki = 0.4
-    robot_state_initial.heading_kd = 0.16
-    robot_state_initial.phase = Phase.TRAVERSE
-    robot_state_initial.state = np.array([[10], [20], [50]])
-    robot_state_initial.battery = 98
-    robot_state_initial.magnetic_field = [0.1, 0.2, 0.3]
-    robot_state_initial.gyro_rotation = [0.5, 0.2, 0.6]
-    robot_state_initial.acceleration = [4.25, 3.2, 0.1]
-    robot_state_initial.is_sim = False
```

Comment:
why did we change is_sim to false? is true a bug

Reply:
Yes o7 i realized that it if we set is_sim to False during the initialization it causes us to run the sensor initialization at the end of `Robot_State`. Since all of the variables in a packet hold relatively the same meaning I should be ok to change the is_sims to True!